code/modules/vehicles/scooter.dm

```diff
@@ -116,24 +129,43 @@
 		return
 
 	var/mob/living/skater = buckled_mobs[1]
-	skater.adjustStaminaLoss(instability*0.5)
-	if (skater.getStaminaLoss() >= 100)
+	skater.adjustStaminaLoss(instability*0.3)
+	if(skater.getStaminaLoss() >= 100)
 		obj_flags = CAN_BE_HIT
 		playsound(src, 'sound/effects/bang.ogg', 20, TRUE)
 		unbuckle_mob(skater)
 		var/atom/throw_target = get_edge_target_turf(src, pick(GLOB.cardinals))
 		skater.throw_at(throw_target, 2, 2)
 		visible_message(span_danger("[skater] loses [skater.p_their()] footing and slams on the ground!"))
-		skater.Paralyze(40)
+		skater.Paralyze(4 SECONDS)
 		grinding = FALSE
 		icon_state = "[initial(icon_state)]"
 		return
 	playsound(src, 'sound/vehicles/skateboard_roll.ogg', 50, TRUE)
-	if(prob(25))
-		var/turf/location = get_turf(src)
-		if(location)
+	var/turf/location = get_turf(src)
+
+	if(location)
+		if(prob(25))
 			location.hotspot_expose(1000,1000)
-		sparks.start() //the most radical way to start plasma fires
+			sparks.start() //the most radical way to start plasma fires
+
+		for(var/mob/living/carbon/victim in location)
+			if(victim.body_position == LYING_DOWN)
+				var/total_brute = victim.getBruteLoss()
+				playsound(location, 'sound/items/trayhit2.ogg', 40)
+				if(victim.stat == DEAD && total_brute >= 175)
+					victim.gib() // Beaten and brutalised, your body gives up
+					playsound(victim, 'sound/effects/splat.ogg', 50)
+				else
+					var/body_part = pick(BODY_ZONE_L_LEG, BODY_ZONE_R_LEG, BODY_ZONE_HEAD, BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_CHEST)
+					victim.apply_damage(damage = 25, damagetype = BRUTE, def_zone = body_part, wound_bonus = 20)
+					victim.Paralyze(1.5 SECONDS)
+					skater.adjustStaminaLoss(instability)
+					victim.visible_message(span_danger("[victim] straight up gets grinded into the ground by [skater]'s [src]! Radical!"))
+		for(var/obj/item/cut_thing in location)
+			var/cut_holder = new/obj/item/knife(null) // The lore implications is that when a skateboard runs over cheese, it summons a knife from the shadow realm to cut it
+			cut_thing.tool_act(skater, cut_holder, TOOL_KNIFE, instant_proccess = TRUE)
+			qdel(cut_holder) // and then disapears the knife back to where it came from.
```

Comment:
This is very hacky and requires you to even add an `instant_process` argument and pass it all around.
Don't. Make use of a proc cutting proc instead which the knife's `tool_act()` also uses.

Reply:
I'll work on it. I'm not sure if it can be implemented exactly as you suggest.

Reply:
I don't know what you mean by "proc cutting proc." 
Are you suggesting I make a new proc that directly sends a `COMSIG_ATOM_TOOL_ACT ` alongside a modification that overrides `time_to_process`? or is that also too hacky?

Reply:
This is a very good case of componentization. What you are after here is not the knife, but the property it holds: cutting.
That is represented by the `tool_behavior` value of `TOOL_KNIFE`.

The way it works is that the knife's `tool_act()` sends the `COMSIG_ATOM_TOOL_ACT` signal, to which the food and similar things register, and they populate a list of the `processing_recipes` available, triggering `process_recipes()`, which you modified.
The problem, and the reason you are having to pass further arguments, is that they all pass the tool as an argument in order to extract the needed information. So that's why you're having to spawn and delete an item.

The proper way to accomplish what you want is to refactor these procs so that instead of passing the whole item by reference, they pass the necessary information, decoupled from it. Then you'll be able to use the tool act without spawning an object.
This is not necessarily a trivial change, as you'd have to refactor down to the `use_tool()` proc,

Consider if you want to make this change. It would be a really good improvement of the code and I can help you through, but, as said, it is not trivial. And if you do it then you can make your balance changes. Code quality comes first and justifies other changes, to me.

If you are not willing to make the change then just remove the cutting feature.

Reply:
Yeah seems easy enough. Cutting feature is non-negotiable and the inspiration of this PR. I will do it.